knowledge-base/print-a-report-directly-at-client-side-without-displaying-it-in-a-viewer.md

```diff
@@ -31,22 +30,29 @@ This article lists the required steps for printing a report directly on the clie
 The existing print functionality in web viewers is based on the following:
 
 1. The report is exported in PDF format;
+
 2. On rendering the PDF document, the reporting engine injects Adobe Javascript that forces the used PDF reader to open its Print dialog;
+
 3. The PDF file is sent to the client in the browser in a hidden frame, where the browser must have properly configured and enabled PDF plugin to load the PDF file in the browser window;
+
 4. Once the PDF is loaded in the browser, hidden from the user, the PDF plugin used as a PDF reader shows its Print dialog.
 
 In the above scenario, the PDF plugin is a must for the browser. The approach is based on PDF export because applications running in a browser do not have permissions to access the machine (technology restrictions). For more details, please check <a href="/html5-report-viewer-direct-print" target="_blank">Printing Reports</a>.
 
-To export the report in a PDF file, you can use a ReportProcessor in code instead of a viewer.
+To export the report in a PDF file, you can use a [ReportProcessor](../t-telerik-reporting-processing-reportprocessor) in code instead of a viewer.

```

Comment:
this is the new API REF link 
https://docs.telerik.com/reporting/api/Telerik.Reporting.Processing.ReportProcessor.html